nixos/modules/config/shells-environment.nix

```diff
@@ -168,9 +168,6 @@ in
 
          ${cfg.extraInit}
 
-         # The setuid/setcap wrappers override other bin directories.
-         export PATH="${config.security.wrapperDir}:$PATH"
-
          # ~/bin if it exists overrides other bin directories.
          export PATH="$HOME/bin:$PATH"
```

Comment:
@0xABAB, That's unrelated to this patch. But if in a separate commit, I don't care. (Although my personal preference is to quote too much than too little.)